src/screens/Article/components/ArticleContent/components/ArticleBody/index.tsx

```diff
@@ -1,19 +1,34 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable react/no-danger */
 import parse from 'html-react-parser';
 import DOMPurify from 'dompurify';
 
 import styles from './styles.module.scss';
 
 interface Props {
   textContent: string;
+  tagList: Array<string>;
 }
 
-function ArticleBody({ textContent }: Props) {
+function ArticleBody({ textContent, tagList }: Props) {
   // eslint-disable-next-line @typescript-eslint/naming-convention
-  const parsedBody = parse(DOMPurify.sanitize(textContent, { ADD_ATTR: ['target'] }));
+  const parsedBody = parse(DOMPurify.sanitize(textContent, { ADD_ATTR: ['target'] })).toString();
 
   return (
     <article className="full-width">
-      <div className={styles.container}>{parsedBody}</div>
+      {tagList.length && (
+        <div className={styles.container}>
+          <div className={styles.contentTags}>
+            <div>Tags: </div>
+            {tagList.map((itemTag, index) => (
+              <div key={index.toString()} className={styles.tag}>
```

Comment:
Dont use index to keys, instead uses the same string.

```suggestion
              <div key={itemTag} className={styles.tag}>
```